references/auth-schemes/pipeline.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/sh
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPTPATH="$( cd "$(dirname "$0")" || exit >/dev/null 2>&1 ; pwd -P )"
+
+proxyName="auth-schemes-v0"
+developerEmail="alien@far-away.space"
+appName="AuthApp"
+productName="APIAuthExamples"
+
+rm -f "$SCRIPTPATH"/jwt.key.pem "$SCRIPTPATH"/jwt.key.pub.pem
+openssl genrsa -out "$SCRIPTPATH"/jwt.key.pem 2048
+openssl rsa -in "$SCRIPTPATH"/jwt.key.pem -pubout -out "$SCRIPTPATH"/jwt.key.pub.pem
+
+_JWT_PRIVATE_KEY=$(cat "$SCRIPTPATH"/jwt.key.pem)
+_JWT_PUBLIC_KEY=$(cat "$SCRIPTPATH"/jwt.key.pub.pem)
+export _JWT_PRIVATE_KEY
+export _JWT_PUBLIC_KEY
+envsubst < "$SCRIPTPATH"/templates/AssignMessage.SetPrivateKey.xml.tmpl > "$SCRIPTPATH"/apiproxy/policies/AssignMessage.SetPrivateKey.xml
```

Comment:
Need to add `gettext` to pipeline-runner

Reply:
same here https://github.com/apigee/devrel/pull/474/files#diff-04cbdbd2cc775a01909a47346f56fde1aceebc2b0248e0e3f4fe6ef439a4e229R27